docs/tex/tutorials/linear-mixed-effects-models.tex

```diff
@@ -0,0 +1,111 @@
+\title{Linear Mixed Effects Models}
+
+\subsection{Linear Mixed Effects Models}
+
+In linear mixed effects models, we wish to model a linear relationship for data points with inputs of varying type, categorized into subgroups, and associated to a continuous output.
+
+We demonstrate with an example in Edward.
+An interactive version with Jupyter notebook is available
+\href{http://nbviewer.jupyter.org/github/blei-lab/edward/blob/master/docs/notebooks/linear_mixed_effects_models.ipynb}{here}.
+
+\subsubsection{Data}
+
+We use the \texttt{InstEval} dataset located \href{https://github.com/blei-lab/edward/blob/master/examples/data/insteval.csv}{here}. It's a dataset of instructor evaluation ratings where our inputs include categories such as \texttt{students} and \texttt{departments}, while our observed variable is the actual rating.
+
+\begin{lstlisting}[language=Python]
+data = pd.read_csv('../../examples/data/insteval.csv')
+data['dcodes'] = data['d'].astype('category').cat.codes
+data['deptcodes'] = data['dept'].astype('category').cat.codes
+data['s'] = data['s'] - 1
+
+train = data.sample(frac=0.8)
+test = data.drop(train.index)
+\end{lstlisting}
+
+\subsubsection{Model}
+
+The main reason we use a mixed model rather than regular linear regression is because we cannot make the independence assumption. Rather, our observations come from sets of groups which can different slopes and intercepts.
+
+For example:
+\begin{itemize}
+\item The observations from a single student are not independent of each other. Rather, some students may systematically give low (or high) lecture ratings.
+\item The observations of a single teachers are not independent from one another. We'd expect good teachers to get generally good ratings and bad teachers to get generally bad ratings.
```

Comment:
> The observations of a single teachers are not independent from one another.

"The observations from a single teacher are not independent of each other."

> We'd expect

"We expect"